db/active_replicator.go

```diff
@@ -313,22 +313,22 @@ func LoadReplicationStatus(dbContext *DatabaseContext, replicationID string) (st
 	}
 
 	pullCheckpoint, _ := getLocalCheckpoint(dbContext, PullCheckpointID(replicationID))
-	if pullCheckpoint != nil {
+	if pullCheckpoint != nil && pullCheckpoint.Status != nil {
```

Comment:
This fix looks good, but I think we should also be setting LastSeqPull and LastSeqPush based on checkpoint.LastSeq when using LoadReplicationStatus, in the case where the unmarshalled status is nil.